app/views/cases/data_requests/edit.html.slim

```diff
@@ -19,7 +19,7 @@ span.visually-hidden
   .column-full
 
     h2.data-request__location.lede= @data_request.location
-    p.data-request__data= @data_request.data
+    p.data-request__request_type= @data_request.request_type
 
```

Comment:
my feeling the old field 'data' is more closer to the new field of "request_type_note",  is it just temporary solution here? as the  "request_type_note" is missing from this page, is it going to be added in another ticket?